cdk/BackendStack.ts

```diff
@@ -59,6 +60,18 @@ export class BackendStack extends Stack {
 			description: 'Role ARN to use in the deploy GitHub Actions Workflow',
 			value: cd.role.roleArn,
 		})
+		const db = new Timestream.CfnDatabase(this, 'db')
+		const table = new Timestream.CfnTable(this, 'table', {
+			databaseName: db.ref,
+			retentionProperties: {
+				MemoryStoreRetentionPeriodInHours: `24`,
+				MagneticStoreRetentionPeriodInDays: '365',
```

Comment:
```suggestion
				MagneticStoreRetentionPeriodInDays: '30',
```

Max 30 days for now on all hello.nrfcloud.com projects.